docs/embed/use-case/serverless/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Serviceless Platforms",
```

Comment:
This is not the use case for embedding.

Reply:
You didn't solve this problem. serverless is not the use case for embedding WasmEdge.

Reply:
I saw your changes below. We should remove this chapter.